app/mailers/patient_mailer.rb

```diff
@@ -10,8 +10,8 @@ def enrollment_email(patient)
 
     # Gather patients and jurisdictions
     # patient.dependents includes the patient themselves if patient.id = patient.responder_id (which should be the case)
-    @patients = patient.active_dependents.uniq.map do |dependent|
-      { patient: dependent, jurisdiction_unique_id: Jurisdiction.find_by_id(dependent.jurisdiction_id).unique_identifier }
+    @patients = patient.active_dependents.uniq.collect do |dependent|
+      { patient: dependent, jurisdiction_unique_id: Jurisdiction.find_by(id: dependent.jurisdiction_id).unique_identifier }
```

Comment:
out of curiosity, are there any performance/behavioral differences between these changes? just wondering

Reply:
not using dynamic find_by is just stylistic and goes with the defaults in the Rails style guide.

i changed the other one back it didn't seem to change anything in terms of rubocop

Reply:
ok thanks!